artbotlib/pipeline_image_names.py

```diff
@@ -0,0 +1,97 @@
+import requests
+import yaml
+from requests_kerberos import HTTPKerberosAuth, REQUIRED
+
+
+def distgit_to_brew(distgit_name, version):
+    brew_name = f"{distgit_name}-container"
+
+    response = requests.get(
+        f"https://raw.githubusercontent.com/openshift/ocp-build-data/openshift-{version}/images/{distgit_name}.yml")
+
+    if response.status_code != 200:
+        print(f"Error: The specified yml file doesn't exist")
+        return "Couldn't find Brew Package name"
+
+    yml_file = yaml.safe_load(response.content)
+    try:
+        component_name = yml_file['distgit']['component']
+    except KeyError:
+        return brew_name
+
+    return component_name
+
+
+def brew_to_cdn(brew_name, variant_name):
+    # Kerberos authentication
+    kerberos_auth = HTTPKerberosAuth(mutual_authentication=REQUIRED, sanitize_mutual_error_response=False)
+
+    try:
+        # Sending the kerberos ticket along with the request
+        response = requests.get(
+            f"https://errata.devel.redhat.com/api/v1/cdn_repo_package_tags?page[number]=1&filter[package_name]={brew_name}&filter[variant_name]={variant_name}",
+            auth=kerberos_auth)
+
+        if response.status_code == 401:
+            print("ERROR: Authentication failure. Enter kerberos password for ticket")
+
+        json_file = response.json()
+        return json_file['data'][0]['relationships']['cdn_repo']['name']
+    except Exception as e:
+        print(f"Error: {e}")
+        return "Couldn't find CDN repo name."
+
+
+def cdn_to_comet(cdn_name):
+    # Kerberos authentication
+    kerberos_auth = HTTPKerberosAuth(mutual_authentication=REQUIRED, sanitize_mutual_error_response=False)
+
+    try:
+        # Sending the kerberos ticket along with the request
+        response = requests.get(f"https://errata.devel.redhat.com/api/v1/cdn_repos/{cdn_name}",
+                                auth=kerberos_auth)
+
+        if response.status_code == 401:
+            print("ERROR: Authentication failure. Kerberos ticket")
+
+        return response.json()['data']['attributes']['external_name']
+    except Exception as e:
+        print(f"Error: {e}")
+        return "Couldn't find delivery repo name."
+
+
+def check_distgit_availability(distgit_repo_name):
```

Comment:
since you're using it like a boolean i'd suggest
```suggestion
def distgit_is_available(distgit_repo_name):
```
or maybe
```suggestion
def distgit_exists(distgit_repo_name):
```

Reply:
Done!